samples/README.md

```diff
@@ -6,75 +6,76 @@ Here you can find all hosted samples of [Web Chat](https://github.com/microsoft/
 
```

Comment:
Only one new entry, but it shifted the table format. Look for the comment below.